topics/variant-analysis/tutorials/aiv-analysis/tutorial.md

```diff
@@ -0,0 +1,404 @@
+>
+>    > <question-title>Question</question-title>
+>    >
+>    > 1. Does everything look ok?
+>    >
+>    > > <solution-title>Answers</solution-title>
+>    > >
+>    > > 1. As expected from the findings so far, the consensus sequence for the HA segment has stretches of N in it, which likely reflect the mapping issues and associated loss of coverage caused by our insufficiently sized collection of references.
+>    > >
+>    > {: .solution}
+>    {: .question}
+>
+{: .hands_on}
+
+# Placing segments on a phylogenetic tree
+
+The next logical step after obtaining the consensus sequences of segments of our sample is to explore how those sequences are related to the sequences in our reference collection.
+To do so, we are going to build multiple sequence alignments (MSAs) per segment of all reference sequences, add our sample segments to those alignments, and build phylogenetic trees, one per segment. We are going to use two rather standard tools, **MAFFT** and **IQTree**, for generating MSAs and trees, respectively.
+
+> <hands-on-title>Exploring phylogeny</hands-on-title>
+>
+> 1. {% tool [MAFFT](toolshed.g2.bx.psu.edu/repos/rnateam/mafft/rbc_mafft/7.508+galaxy0) %}
+>    - {% icon param-collection %} *"Sequences to align"*: `References per segment (INSAFlu)`
+>    - *"Data type"*: `Nucleic Acids`
+>    - *"Matrix selection"*: `No matrix`
+>
+>    The result is a collection of MSAs, each representing all reference sequences of one genome segment.
+>
+> 2. {% tool [MAFFT add](toolshed.g2.bx.psu.edu/repos/rnateam/mafft/rbc_mafft_add/7.508+galaxy0) %}
+>    - {% icon param-collection %} *"Sequences to add to the alignment"*: consensus sequences with simplified names; output of **Replace**
+>    - {% icon param-collection %} *"Alignment"*: collection of MSAs; output of MAFFT
+>    - *"What do you want to add to the alignment"*: `A single sequence`
+>    - *"Keep alignment length"*: `No`
+>
+>    The result is a new collection with each of our sample consensus sequences added to the respective segment MSA.
+>
+> 3. {% tool [IQTree](toolshed.g2.bx.psu.edu/repos/iuc/iqtree/iqtree/2.1.2+galaxy2) %}
+>    - {% icon param-collection %} *"Specify input alignment file in PHYLIP, FASTA, NEXUS, CLUSTAL or MSF format."*: output of **MAFFT add**
+>    - *"Specify sequence type ..."*: `DNA`
+>
+{: .hands_on}
+
+# Conclusion
+
+Analysis workflows for influenza whole-genome sequencing data need to take into account the specific characteristics of the viral genome. Due to their higher natural variability this is especially true for avian influenza samples and for the HA- and NA-encoding segments of the genome.
+
+Nevertheless, it is possible, with a carefully chosen collections of reference segment sequences and bioinformatic tools, to avoid a computaionally expensive de-novo assembly approach and to use mapping against a dynamically compiled reference genome instead.
```

Comment:
```suggestion
Nevertheless, it is possible, with a carefully chosen collections of reference segment sequences and bioinformatic tools, to avoid a computationally expensive de-novo assembly approach and to use mapping against a dynamically compiled reference genome instead.
```